contrib/CHRU2014/paper_averaging.py

```diff
@@ -42,7 +42,7 @@
 
 import burnman
 from burnman import minerals
-import misc.colors as colors
+import .colors as colors
```

Comment:
I think `from contrib.CHRU2014 import colors` would work (here and in other scripts).